regparser/notice/build.py

```diff
@@ -320,6 +320,11 @@ def process_amendments(notice, notice_xml):
             create_xmlless_changes(other_labels, notice_changes)
 
             for cfr_part, rel_labels in labels_by_part.iteritems():
+            # labels_for_part = {part: labels
+            #                    for part, labels in labels_by_part.iteritems()
+            #                    if part == default_cfr_part}
+            #
+            # for cfr_part, rel_labels in labels_for_part.iteritems():
```

Comment:
D'OH. I made this change so that I could parse X faster. I'll have to revert it.